alibi_detect/utils/tests/test_distance.py

```diff
@@ -1,10 +1,8 @@
-import dask.array as da
 import numpy as np
 from scipy.spatial.distance import cityblock
 from itertools import product
 import pytest
-from alibi_detect.utils.distance import (pairwise_distance, maximum_mean_discrepancy, abdm,
-                                         cityblock_batch, mvdm, multidim_scaling, relative_euclidean_distance)
+from alibi_detect.utils.distance import pairwise_distance, abdm, cityblock_batch, mvdm, multidim_scaling
```

Comment:
Are the `dask` array tests not really required anymore?

Reply:
No, `dask` is no longer supported.